### DISCUSSION TITLE
TVS channel on malaysia

### DISCUSSION THREAD

**Discussion by DDesmond95** [iptv-org/community#172] [Q&A] [answered]
I remember there's TVS on the list, but now it's gone?
I tried using different app to check, and all of it don't show it, while it still in the code.
So what's the problem?

**Comment by heroku-miraheze** [comment]
The problem is for cache, so, please clear cache and try again, and if the problem persists, you can try uninstalling the app and installing the app again. This solutions helps you. 👍

**Comment by DDesmond95** [comment]
But it's freshly install app so theres no cache also

**Comment by Sphinxroot** [comment]
hello are you talking  about this site https://rtmklik.rtm.gov.my/  or another one ? if its the the one   referer seem to have change to fit channel number

#EXTINF:-1,TV1
https://rtmlive01.secureswiftcontent.com/rtmchannel/01-manifest.m3u8|Referer=https://rtmklik.rtm.gov.my/tv/1

or

#EXTINF:-1,TV1
#EXTVLCOPT:http-referrer=https://rtmklik.rtm.gov.my/tv/1
https://rtmlive01.secureswiftcontent.com/rtmchannel/01-manifest.m3u8

and will be changed for all 8 off them

**Reply by BellezaEmporium** [reply]
> 
> 
> hello are you talking about this site https://rtmklik.rtm.gov.my/ or another one ? if its the the one referer seem to have change to fit channel number
> 
> #EXTINF:-1,TV1 [https://rtmlive01.secureswiftcontent.com/rtmchannel/01-manifest.m3u8|Referer=https://rtmklik.rtm.gov.my/tv/1](https://rtmlive01.secureswiftcontent.com/rtmchannel/01-manifest.m3u8%7CReferer=https://rtmklik.rtm.gov.my/tv/1)
> 
> or
> 
> #EXTINF:-1,TV1 #EXTVLCOPT:http-referrer=https://rtmklik.rtm.gov.my/tv/1 https://rtmlive01.secureswiftcontent.com/rtmchannel/01-manifest.m3u8
> 
> and will be changed for all 8 off them

If possible, make a PR containing the referrer changes.

**Reply by DDesmond95** [reply]
I'm talking about TVS, which got before. But now it's gone. 

They got this website if that's help
https://tvstv.my/live/

**Reply by DDesmond95** [reply]
I think I found the problem.

When I use https://raw.githubusercontent.com/iptv-org/iptv/master/channels/my.m3u, it got TVS
But when using this https://iptv-org.github.io/iptv/countries/my.m3u, then no TVS.
Not sure why this happen

**Reply by BellezaEmporium** [reply, marked as answer]
It's because link doesn't work, simply. Every offline channel do not appear on the github.io playlists.

**Reply by DDesmond95** [reply]
Umm, I tried myself there are 2 channel that say offline there are totally watchable.
1. Astro Vaanavil
2. TVS

Just that it's not 24/7 only.

**Comment by BellezaEmporium** [comment]
That might also be because you're testing them from a Malaysian IP.

**Reply by DDesmond95** [reply]
Well yeah... So can you guys fix it?
